vlplidar_ws/scripts/lidar-slam-bringup.sh

```diff
@@ -4,10 +4,10 @@ source $ROS2_WS/install/setup.bash
 
 {
     ros2 launch vlp_cartographer vlp_driver.launch.py
-} <&0 &
+} &
 
 {
     ros2 launch vlp_cartographer cartographer_demo.launch.py using_odom:=False
-} <&0
+}
```

Comment:
Everything looks good to me, just a small comment:

<hr>

I think this can be simplified to:

```sh
ros2 launch vlp_cartographer cartographer_demo.launch.py using_odom:=False
```

Since the README tells us to always manually launch the VLP driver